src/Simplify.elm

```diff
@@ -3605,6 +3719,228 @@ listIndexedMapChecks { lookupTable, fnRange, firstArg } =
                     []
 
 
+listFoldlChecks : CheckInfo -> List (Error {})
+listFoldlChecks checkInfo =
+    listFoldAnyDirectionChecks "foldl" checkInfo
+
+
+listFoldrChecks : CheckInfo -> List (Error {})
+listFoldrChecks checkInfo =
+    listFoldAnyDirectionChecks "foldr" checkInfo
+
+
+listFoldAnyDirectionChecks : String -> CheckInfo -> List (Error {})
+listFoldAnyDirectionChecks foldOperationName checkInfo =
+    -- TODO discuss adding
+    -- List.fold (&|) initial list --> initial &| (List.all identity list)
+    --         and piping versions
+    -- List.fold (&|) initial --> (List.all identity >> (&|) initial)
+    --     Are those simplifications? arguable, I'm leaning no
```

Comment:
No for this one. I'd say we never want to introduce `>>` or `<<`, because that is often a bit hard to read for beginners.

EDIT: You said the same thing below